src/InteractionModNborhood.H

```diff
@@ -54,7 +55,8 @@ static void binaryInteractionNborhood ( const int a_i, /*!< Index of infectious
         }
     }
 
-    Gpu::Atomic::Multiply(&a_prob_ptr[a_j], prob);
+    AMREX_ALWAYS_ASSERT(a_j == a_thread);
```

Comment:
should we change these to a regular `AMREX_ASSERT` so that it only happens in debug mode? There's a performance hit associated with the assert, too (could even be more than the atomic)...

Reply:
I switched to `AMREX_ASSERT`. I did not observe a difference in the small runs that I do to test changes, but hopefully it speeds things up for larger runs.